prusti-common/src/vir/ast/expr.rs

```diff
@@ -292,7 +302,7 @@ impl fmt::Display for BinOpKind {
             BinOpKind::BitOr => write!(f, "|"),
             BinOpKind::BitXor => write!(f, "^"),
             BinOpKind::Shl => write!(f, "<<"),
-            BinOpKind::Shr => write!(f, ">>"),           
+            BinOpKind::LShr| BinOpKind::AShr => write!(f, ">>"),           
```

Comment:
It is a bit confusing that two different operators are displayed in the same way. One of them could be `>>>` like in Java.